planning/autoware_obstacle_stop_planner/schema/obstacle_stop_planner.schema.json

```
@@ -0,0 +1,322 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter for obstacle stop planner",
+  "type": "object",
+  "definitions": {
+    "obstacle_stop_planner": {
+      "type": "object",
+      "properties": {
+        "chattering_threshold": {
+          "type": "number",
+          "description": "INSERT_DESCRIeven if the obstacle disappears, the stop judgment continues for chattering_threshold [s]PTION",
+          "default": "0.5"
+        },
+        "max_velocity": {
+          "type": "number",
+          "description": "max velocity [m/s]",
+          "default": "20.0"
+        },
+        "ego_nearest_dist_threshold": {
+          "type": "number",
+          "description": "[m]",
+          "default": "3.0"
+        },
+        "ego_nearest_yaw_threshold": {
+          "type": "number",
+          "description": "[rad] = 60 [deg]",
+          "default": "1.046"
+        },
+        "enable_slow_down": {
+          "type": "boolean",
+          "description": "whether to use slow down planner [-]",
+          "default": "False"
```

Could you make it to lower case?
It seems it causes some error when converting to markdown if false/true are capitalized.

```suggestion
          "default": "false"
```